src/registry_schemas/schemas/business.json

```diff
@@ -11,6 +11,16 @@
       "default": "",
       "examples": ["legal_name"],
       "pattern": "^(.*)$"
+    },
+    "legalType": {
+      "type": "string",
+      "title": "The Legal Type of the business.",
+      "default": "BC",
+      "enum": [
```

Comment:
Should we add the list of corp types, as alterations can be done from most of them?

Reply:
sure